homeassistant/components/ozw/light.py

```diff
@@ -140,29 +140,39 @@ def color_temp(self):
     def async_set_duration(self, **kwargs):
         """Set the transition time for the brightness value.
 
-        Zwave Dimming Duration values:
-        0       = instant
-        0-127   = 1 second to 127 seconds
-        128-254 = 1 minute to 127 minutes
-        255     = factory default
+        Zwave Dimming Duration values now use seconds as an
+        integer (max: 7620 seconds or 127 mins)
         """
         if self.values.dimming_duration is None:
             return
 
+        ozw_version = self.values.primary.ozw_instance.get_status().openzwave_version
+        ozw_build = int(ozw_version.split(".")[2])
+        ozw_major = float(ozw_version[0:1])
+
         if ATTR_TRANSITION not in kwargs:
             # no transition specified by user, use defaults
-            new_value = 255
+            if ozw_build >= 1205 and ozw_major >= 1.6:
```

Comment:
We should check major and minor version.

Reply:
i just called it 'major' 1.6 is the `major.minor`

Reply:
If you'd like I could rename the variable to something like `ozw_check_ver` ?